README.md

```diff
@@ -76,7 +76,24 @@ _ie. Icons/icons/House.vue_
 4) Look for any output errors in terminal - if there's a file name with a space, or the xml tag remains in the SVG file it won't compile properly.
 5) You can now add "Full Stack Developer" to your resume ;)
 
+### Photon v2 components.
 
+Photon v2 components are a new system of styling photon components. 
+
+In order to get the theme files to load you need to create a loader that pulls in all the theme files.
+
+run the following command
+```typescript
+cp themeLoader.sample.js themeLoader.js
```

Comment:
do you mean
`themeLocation.sample.js`

Reply:
Correc. I've updated this piece a bit since this comment.

Reply:
It's also bash :D. not Typescript!